### ISSUE TITLE
online search isn't working :(

### ISSUE THREAD

**Issue by 0xMRTT** [mjakeman/extension-manager#253]
![image](https://user-images.githubusercontent.com/105598867/188503092-0c78fed7-2f2a-4011-8cf4-d16cc20d16e3.png)
![Screenshot from 2022-09-05 21-01-21](https://user-images.githubusercontent.com/105598867/188503111-954b04b7-8ca0-494f-9030-4d3d3b2bccc4.png)

I use arch, with the browser connector and the official gnome app

**Comment by mjakeman** [comment]
Odd. Is it still an issue now, rather than a temporary outage?

It's probably some networking related issue we don't have debug output for right now. That will be fixed in #31. In the mean time, do you have any special network configuration like Proxy, VPN, custom DNS, etc?

**Comment by 0xMRTT** [comment]
I still have the issue. I'm using Cloudflare

**Comment by JamiKettunen** [comment]
I just started seeing this on [the v0.4.0 flatpak](https://flathub.org/apps/com.mattjakeman.ExtensionManager) under [Chimera Linux](https://chimera-linux.org/):
```
$ flatpak info com.mattjakeman.ExtensionManager

Extension Manager - Browse, install, and manage GNOME Shell Extensions.

          ID: com.mattjakeman.ExtensionManager
         Ref: app/com.mattjakeman.ExtensionManager/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 0.4.0
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 1.9 MB
     Runtime: org.gnome.Platform/x86_64/43
         Sdk: org.gnome.Sdk/x86_64/43

      Commit: d51a72124b1247432b3bbe255e881864abacb7d20aa309f86427f057df116f77
      Parent: 562b537ed281af5deb5dda043681735c0a6d7d0630b2499b0b4a9d6c08f04bb9
     Subject: Update extension-manager to version 0.4.0 (c6150d6f)
        Date: 2022-10-06 11:53:57 +0000
```
For what it's worth I also have Chimera in a VM on another system where I tested #14 and it currently doesn't have this problem and is able to search just fine.

**Comment by JamiKettunen** [comment]
And now it works again all of the sudden so the flatpak runtime version certainly had nothing to do with it at least (still would be nice to get the update in), odd issue. I didn't see anything interesting in `flatpak run com.mattjakeman.ExtensionManager` output when it didn't work either.

**Comment by mjakeman** [comment]
Most/all of our search problems should be fixed as of v0.4.3 - please file a new issue if this is still happening.
